src/ui/components/atoms/badges/badge.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+$ads-color-violet: #9a4aff;
+$ads-color-blue: #180bd4;
```

Comment:
Please check if these tokens are already in the globals; if not, please put them there